lib/kitchen/composite_page_element.rb

```diff
@@ -43,13 +43,21 @@ def is_index?
       has_class?('os-index-container')
     end
 
-    # Returns true if this page is a book reference
+    # Returns true if this page is a book citation reference
     #
     # @return [Boolean]
     #
-    def is_reference?
+    def is_citation_reference?
       has_class?('os-reference-container')
     end
 
+    # Returns true if this page is a book section reference
+    #
+    # @return [Boolean]
+    #
+    def is_section_reference?
+      has_class?('os-references-container')
+    end
+
```

Comment:
I checked and class `reference` is used in books that contain references in form of citation, similar to footnotes:

![image](https://user-images.githubusercontent.com/40228854/121500678-d184dc00-c9de-11eb-950e-e2b8824dfe24.png)

While class `references` is used when book contain a whole section in Introduction page:

![image](https://user-images.githubusercontent.com/40228854/121501363-769fb480-c9df-11eb-901d-103dfa1d38b0.png)

Reply:
The one letter difference is so unfortunate 😂

Reply:
Maybe it would be good to add the difference that you explained in this comment to the file

Reply:
Yes, sure :slightly_smiling_face: It will be less confusing in the future. I added needed comment.